saviSimulation/savi/asl/demo.asl

```diff
@@ -5,5 +5,6 @@
  */
 
 //{include("lostInThought.asl")}
-{include("working.asl")}
+{include("talkative.asl")}
+//{include("working.asl")}
```

Comment:
I see that the talkative.asl agent is able to communicate. This agent doesn't otherwise perform useful behaviour. That said, it may be good as an example for which to develop the final behaviour.